components/card.css

```diff
@@ -24,11 +29,81 @@
 .card-title {
 	margin: 1rem 0;
 }
+/* CARD PRICE FOR ECOM */
 .card-price-striked {
 	text-decoration: line-through;
 	margin: 1rem;
-	/* color: #ccc; */
+	font-size: 1.3rem;
+	color: var(--color-grey-dark);
 }
 .card-price {
-	font-size: 1.5rem;
+	font-size: 1.6rem;
+	color: var(--color-primary);
+}
+
+/* BLOG CARD */
+.card-container {
+	display: flex;
+	flex-direction: column;
+	margin: 1rem;
+	padding: 1rem;
+	width: 36rem;
+	position: relative;
+	background-color: var(--color-white);
+	-webkit-box-shadow: -1px 1px 15px 0px rgba(176, 176, 176, 1);
+	-moz-box-shadow: -1px 1px 15px 0px rgba(176, 176, 176, 1);
+	box-shadow: -1px 1px 15px 0px rgba(176, 176, 176, 1);
+}
+.img-box,
+img {
+	width: 100%;
+}
+
+.card-heading-primary {
+	margin: 0;
+}
+.card-heading-sub {
+	color: var(--color-grey-dark);
+	margin: 0;
```

Comment:
in line 63 and 67 the css rule declarations are same. Hence, advised to use a variable instead.